Error while checking build/O2Physics/code-check for 4b116bbb3d1f9999253990df7ee7f47b70e9a961 at 2026-08-19 19:16:
```
## sw/BUILD/lhapdf-latest/log
collect2: error: ld returned 1 exit status


## sw/BUILD/O2-latest/log
2026-08-19T18:05:03.023+0200 [81185:140708278668928] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.043+0200 [81185:140708278668928] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.025+0200 [81185:140708278668928] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.264+0200 [81248:140512322240128] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.264+0200 [81248:140512322240128] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.266+0200 [81248:140512322240128] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.412+0200 [81184:139740719574656] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.412+0200 [81184:139740719574656] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.413+0200 [81184:139740719574656] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.477+0200 [81186:140651328921216] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.477+0200 [81186:140651328921216] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.478+0200 [81186:140651328921216] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.758+0200 [81247:139895063916160] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.758+0200 [81247:139895063916160] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.759+0200 [81247:139895063916160] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.858+0200 [81249:140534524149376] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:03.858+0200 [81249:140534524149376] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:23.860+0200 [81249:140534524149376] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.220+0200 [81390:139783795592832] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.220+0200 [81390:139783795592832] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:24.221+0200 [81390:139783795592832] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.381+0200 [81251:140366677757568] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.381+0200 [81251:140366677757568] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:24.382+0200 [81251:140366677757568] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.554+0200 [81252:139812519001728] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.554+0200 [81252:139812519001728] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:24.555+0200 [81252:139812519001728] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.908+0200 [81389:140187871723136] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.909+0200 [81389:140187871723136] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:24.910+0200 [81389:140187871723136] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.950+0200 [81391:140258432425600] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.950+0200 [81391:140258432425600] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:24.951+0200 [81391:140258432425600] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:05.817+0200 [81472:139687069576832] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:05.817+0200 [81472:139687069576832] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:25.818+0200 [81472:139687069576832] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.793+0200 [81250:140242864078464] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:04.793+0200 [81250:140242864078464] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:44.794+0200 [81250:140242864078464] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:05.952+0200 [81474:140638995734144] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:05.952+0200 [81474:140638995734144] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
2026-08-19T18:05:45.955+0200 [81474:140638995734144] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "FindMissingBlobs()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:06.222+0200 [81473:140541154492032] [buildboxcommon_grpcretrier.cpp:177] [ERROR] Retry limit (0) exceeded for "ActionCache.GetActionResult()", last gRPC error was [14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame]
2026-08-19T18:05:06.222+0200 [81473:140541154492032] [executioncontext.cpp:544] [ERROR] Error while querying action cache at "http://localhost:8980": 14: failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:8980: connection attempt timed out before receiving SETTINGS frame
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/17509/4b116bbb3d1f9999253990df7ee7f47b70e9a961/build_O2Physics_code-check/pretty.html).

--- a/PWGCF/Flow/Tasks/pidFlowPtCorr.cxx
+++ b/PWGCF/Flow/Tasks/pidFlowPtCorr.cxx
@@ -3363,7 +3363,7 @@ struct PidFlowPtCorr {
 
       float weight = 1.f;
       setParticleNUEWeight(weight, track, cent, pid);
-      const std::size_t index = static_cast<std::size_t>(pid - MyParticleType::kPion);
+      const auto index = static_cast<std::size_t>(pid - MyParticleType::kPion);
       const double particleWeight = weight;
       const double weight2 = particleWeight * particleWeight;
       const double pt = track.pt();

